fe/fe-core/src/main/java/org/apache/doris/statistics/StatsDeriveResult.java

```diff
@@ -146,4 +147,22 @@ public StatsDeriveResult merge(StatsDeriveResult other) {
     public StatsDeriveResult copy() {
         return new StatsDeriveResult(this);
     }
+
+    public StatsDeriveResult updateRowCountOnCopy(double selectivity) {
+        StatsDeriveResult copy = new StatsDeriveResult(this);
+        copy.setRowCount(rowCount * selectivity);
+        for (Entry<Slot, ColumnStat> entry : copy.slotToColumnStats.entrySet()) {
+            entry.getValue().updateBySelectivity(selectivity, rowCount);
```

Comment:
this logic is not solid. 
select A , B from T where B=1.
if the selectivity is 0.1, there is no reason to say that A.ndv reduced by 10 times.